setupext.py

```diff
@@ -17,7 +17,8 @@
 import versioneer
 
 
-PY3 = (sys.version_info[0] >= 3)
+PY3 = (sys.version_info.major >= 3)
+PY32min = (PY3 and sys.version_info.minor >= 2)
```

Comment:
This would be broken on 4.0 (though it's anyone's guess as to whether the API would be the same anyway.)